weather_mv/loader_pipeline/pipeline_test.py

```diff
@@ -50,6 +56,10 @@ def setUp(self) -> None:
                 "d": {"dims": ("a",), "data": [3.0]},
             }
         }
+        self.test_single_var = xr.open_dataset(
```

Comment:
This may not work on machines that do not have the grib backend setup.

Reply:
Fair point -- I'll use our Grib backend handler.